lib/record_store/record/a.rb

```diff
@@ -26,5 +30,11 @@ def valid_address?
     rescue IPAddr::InvalidAddressError
       errors.add(:address, 'is invalid')
     end
+
+    def normalize_address(value)
+      IPAddr.new(value).to_s
+    rescue IPAddr::InvalidAddressError
+      value
```

Comment:
Is there a reason why we're returning value when we get an `InvalidAddressError` (instead of raising an error)

Reply:
In that case, you could remove the 
```rescue IPAddr::InvalidAddressError
      value```
from the normalize function then...